Rovers/Rovers.tex

```diff
@@ -0,0 +1,249 @@
+It would be beneficial to our members to get some experience in this area.
+A rover would look super good for SPEX at Imagine RIT. 
+The rover would be rather large and would attract many eyes. We could even demo it outside if there is sufficient space. 
+A rover would be very easy to get video and photos of for SPEX promotions. 
+Having a rover is also another opportunity for SPEX to fundraise. 
+There is plenty of space to place company logos on the body of the rover. 
+It would also allow for SPEX to reach out to robotics companies.
+The self driving component would be the heaviest computer science project SPEX would have attempted, this would help with retention of CS and SE majors. 
+Machine learning and artificial intelligence are at the forefront of computer science right now. 
+They are heavily desired in industry including space exploration.
+
+% Subsections
+
+\subsection{Mindset}
+\label{subsec:mindset}
+The purpose of this project is to investigate and implement a mock rover.
+Because of this the team members must be in the mindset to analyze each part of the project and identify as many problem areas that need answers as possible. 
+This means being specific on how we are going to accomplish our goals. 
+What material, what algorithm, with what method will we be accomplishing this.
+It is very easy to over estimate the team's ability especially when the project is so new to the team.
+
+\subsection{Traceability}
+\label{subsec:traceability}
+
+It is important that the team members document the sources they use to gather information. 
+There will be a Google Drive folder that will hold notes with links to any books, articles, media or their sources that are relevant to the rover project. 
+The software and hardware will be tracked on GitHub. 
+This includes the technical report.
+
+\subsection{Assessing Technical Skills}
+\label{subsec:assessing-technical-skills}
+
+Building a rover, especially one built to URC specifications requires a large team with a diverse set of technical skills. 
+This project will require SPEX to look at and asses what areas are sufficient and what areas need to be developed in order to build a rover. 
+It will also let us know to what spec we are currently capable of building such a rover. 
+
+\subsection{Accessibility}
+\label{subsec:plug-n-play}
+
+The project would be very ambitious. It would require at a minimum a handful (4-5) members each with different areas of expertise. 
+The members would need that knowledge as well as \LaTeX{}, notetaking, and good research practices. 
+This would be to deliver the MVP.
+A larger team would be able to accomplish more.  
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project would involve divvying out the different project areas to the different team members or small groups and each week meeting to discuss progress. 
```

Comment:
I would make this a high level summary of each deliverable overall. Then you can go into details in the `Deliverables` subsection